doc/source/index.rst

```diff
@@ -251,13 +251,6 @@ produces:
 
     print("hello, world!", file=sys.stderr)
 
-.. note::
-  To adjust the CSS of the ``stderr`` stream, use the ``stderr`` class. If you are using
-  the default Sphinx theme, for example, add the following
-  `custom CSS <https://alabaster.readthedocs.io/en/latest/customization.html#custom-stylesheet>`_:
-    ``.stderr {background-color: #FCC}``
-
-
```

Comment:
I agree, and I have added preliminary notes in the css file. I would like to make the styles a bit more fined grained, so there are classes for language, if possible, so if people want to crazy with the CSS they can. Anyone who is styling it will just deal with the webpage, but we could link to the CSS file. I don't know if it warrants specific space in the docs, other thank a link, but we could do that. It's not quite the same as Jupyter because it uses a different library for line numbering. Perhaps giving a quick example of adding additional CSS would be useful.

Reply:
Ok I have added a short "Styling options" section to the docs.  Let me know if it makes sense. I will not add anything further regarding language, as that could be handled by overriding Pygments.

Reply:
Thanks, looks good.